code/src/components/GameScreen.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react'
+import { useSelector, useDispatch } from 'react-redux'
+
+import { continueGame } from '../reducers/games'
+
+
+const GameScreen = () => {
+  const actions = useSelector(store => store.games.actions)
+  const dispatch = useDispatch()
+
+  const onButtonclick = (item) => {
+    dispatch(continueGame(item))
+    console.log(item)
+  }
+
+  console.log(actions)
+  return (
+    <section className='game-on'>
+      <div className='head-description'>
+        <h2>{actions.description}</h2> 
+      </div> 
+      <section className='direction-wrapper'>
+            {actions.actions.map(item => (
+        <>
```

Comment:
remember that every element created by iteration needs a key.

You have only one immediate child element (div) so you shouldn't use react fragment here.